proto/storage/vulnerability.proto

```diff
@@ -56,6 +56,7 @@ message EmbeddedVulnerability {
   // cvss_metrics stores list of cvss scores from different sources like nvd, Redhat etc
   repeated CVSSScore cvss_metrics = 21;
   float nvd_cvss = 22; // @gotags: search:"NVD CVSS,store"
+  EPSSScore epss_score=23;
```

Comment:
Let’s brainstorm on the naming.
* **EPSS score** is the most common user-facing text for the (probability) score
* The percentile (rank) is not a score.

Two thoughts:
* Is `epss` too terse for the property? It seems potentially intuitive like `epss.epssToBeDetermined`
* On other side of the coin, is `EPSS` reasonable for the corresponding message type?

Reply:
I believe EPSS is a good choice. It is similar to CVSSV2 which represents all fields related to cvss like score, severity etc. scanner is using the name EPSS in their api proto as well so it will be consistent.

Reply:
+1 to naming the type and field to EPSS.